app/javascript/lib/theme.js

```diff
@@ -5,7 +5,9 @@ export const colors = {
   lightPink: '#FFA7C2',
   pink: '#FE6B6E',
   teal: '#5fc19f',
+  lightYellow: '#FAE7A7',
   yellow: '#fbd273',
+  yellow2: '#CDBD64',
```

Comment:
`yellow2` doesn't really tell us much so I'd like to avoid. I'm curious, where do we use `pink`, `yellow`, etc through-out the application? Are they actually shared color codes or are the specific to a single component? Would it make more sense to define those colors in their specific components rather than co-locating them?

Reply:
`pink` & `yellow` are both used in `panel.jsx` (for styling the slack and github information) and `pink` is also used in `semi-circle.jsx` to style the pvd buildings. `blue` and `lightBlue` are only used in the semi-circle component. I think it makes sense to define `blue`,`lightBlue`,`yellow`, and `yellow2` (with a more descriptive name) in their specific components and leave `pink` as shared. What do you think about that?

Reply:
Sounds good to me.